en/docs/references/concepts/authorization/jarm.md

```diff
@@ -0,0 +1,177 @@
+# JWT Secured Authorization Response Mode (JARM) for OAuth 2.0
+
+The [JWT Secured Authorization Response Mode for OAuth 2.0 (JARM) specification](https://openid.net/specs/openid-financial-api-jarm-ID1.html) defines new JWT-based modes to encode OAuth2 authorization responses. These modes allow clients to request authorization response parameters and additional data in JWT format.
+
+## Authorization flow
```

Comment:
Shall we update this as `JARM authorization flow`

Reply:
+1